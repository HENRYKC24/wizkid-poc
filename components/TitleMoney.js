import React from "react";
import { StyleSheet, Text, View } from "react-native";

const TitleMoney = () => {
  return (
    <View style={styles.container}>
      <Text style={styles.title}>SIMPLE FREE ACCOUNT</Text>
      <Text style={styles.balance}>$1,560.20</Text>
    </View>
  );
};

export default TitleMoney;

const styles = StyleSheet.create({
  container: {
    marginTop: 10,
    alignItems: "center",
  },
  title: {
    fontWeight: "500",
    color: "#000",
    fontSize: 14,
  },
  balance: {
    fontWeight: "700",
    color: "#000",
    fontSize: 32,
    letterSpacing: 0.02,
  },
});
