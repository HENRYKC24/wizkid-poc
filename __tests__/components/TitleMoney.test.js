import React from "react";
import renderer from "react-test-renderer";

import TitleMoney from "../../components/TitleMoney";

describe("<TitleMoney />", () => {
  it("has 1 child", () => {
    const tree = renderer.create(<TitleMoney />).toJSON();
    expect(tree.children.length).toBe(2);
  });

  it("renders correctly", () => {
    const tree = renderer.create(<TitleMoney />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
