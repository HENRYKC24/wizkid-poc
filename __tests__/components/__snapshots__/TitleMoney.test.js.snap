// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TitleMoney /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "marginTop": 10,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#000",
        "fontSize": 14,
        "fontWeight": "500",
      }
    }
  >
    SIMPLE FREE ACCOUNT
  </Text>
  <Text
    style={
      Object {
        "color": "#000",
        "fontSize": 32,
        "fontWeight": "700",
        "letterSpacing": 0.02,
      }
    }
  >
    $1,560.20
  </Text>
</View>
`;
